x-pack/plugins/observability_solution/infra/public/pages/metrics/hosts/components/tabs/alerts_tab_badge.tsx

```diff
@@ -7,15 +7,17 @@
 import React from 'react';
 import { EuiIcon, EuiLoadingSpinner, EuiBadge, EuiToolTip } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
+import { AlertConsumers } from '@kbn/rule-data-utils';
+import { INFRA_RULE_TYPE_IDS } from '../../../../../../common/alerting/metrics/types';
 import { useAlertsCount } from '../../../../../hooks/use_alerts_count';
 import { useAlertsQuery } from '../../hooks/use_alerts_query';
-import { infraAlertFeatureIds } from '../../../../../components/shared/alerts/constants';
 
 export const AlertsTabBadge = () => {
   const { alertsEsQuery } = useAlertsQuery();
 
   const { alertsCount, loading, error } = useAlertsCount({
-    featureIds: infraAlertFeatureIds,
+    ruleTypeIds: INFRA_RULE_TYPE_IDS,
+    consumers: [AlertConsumers.INFRASTRUCTURE, AlertConsumers.ALERTS, AlertConsumers.OBSERVABILITY],
```

Comment:
Same here, shouldn't it be `INFRA_ALERT_CONSUMERS`?

Reply:
Good point. I missed it. Fixed in [`7758052` (#183756)](https://github.com/elastic/kibana/pull/183756/commits/77580527b3a678aa5b9c5cc5942b8122be628aee).